desc/io/input_reader.py

```diff
@@ -776,6 +778,166 @@ def write_desc_input(filename, inputs, header=""):  # noqa: C901 - fxn too compl
 
         f.close()
 
+    @staticmethod
+    def descout_to_input(  # noqa: C901 - fxn too complex
+        filename,
+        inputs,
+        header="#DESC-generated input file",
+        ftol=1e-2,
```

Comment:
these arguments aren't included in the docstring, we should explain how they're used.

Reply:
Added short description.